wrapper/CSharp/wolfSSL-Example-IOCallbacks/wolfSSL-Example-IOCallbacks.cs

```diff
@@ -135,6 +135,67 @@ public static uint my_psk_server_cb(IntPtr ssl, string identity, IntPtr key, uin
         return (uint)4;
     }
 
+    /// <summary>
+    /// Example of a certificate verify function
+    /// </summary>
+    /// <param name="preverify"></param>
+    /// <param name="store">pointer to a WOLFSSL_X509_STORE_CTX</param>
+    /// <returns>size of key set</returns>
+    public static int my_verify_cb(int preverify, IntPtr store)
+    {
+        if (store == IntPtr.Zero)
+        {
+            Console.WriteLine("store is null");
+        }
+
+        Console.WriteLine("Status of certificate verify = " + preverify);
+
+        /* look at the current cert in store */
+        try
+        {
+
+            X509 x509 = wolfssl.X509_STORE_CTX_get_current_cert(store);
+
+
+            Console.WriteLine("Issuer : " + x509.Issuer);
+            Console.WriteLine("Subject : " + x509.Subject);
+
+            Console.WriteLine("PEM of certificate:");
+            Console.WriteLine(System.Text.Encoding.UTF8.GetString(x509.Export()));
+
+            Console.WriteLine("DER of certificate:");
+            Console.WriteLine(BitConverter.ToString(x509.Export(wolfssl.SSL_FILETYPE_ASN1)));
+
+            Console.WriteLine("Public key:");
+            Console.WriteLine(BitConverter.ToString(x509.GetPublicKey()));
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine("Unable to get X509's");

```

Comment:
I get `e` unused here. Can we add `e.ToString()` here and also below at line 193?

Reply:
added